plus/tool/src/main/webapp/WEB-INF/templates/form.html

```diff
@@ -182,6 +182,19 @@ <h1><span th="#{plus.tool.tenant.title}">Tenant:<span>
     let inp = document.getElementById(id+'-boolean');
     inp.value = checked ? 'true' : 'false';
 }
+</script>
+
+<script th:inline="javascript">
+    function validateTenantForm() {
```

Comment:
I think you should have a namespace for the plus javascript functions. You could just create a window scope object called "plus" and add your functions to that. You would have to ensure that your onsubmit calls "plus.validateTenantForm" though.

Reply:
Thanks for the suggestion! I would like to remove the script and use ```pattern-attribute``` as Kunal mentioned in this case.